plugins/woocommerce/client/blocks/assets/js/utils/schema-parser/index.ts

```diff
@@ -9,13 +9,17 @@ const ajv = new Ajv( {
 	allErrors: true,
 	$data: true,
 	validateSchema: true,
-	validateFormats: false,
+	validateFormats: true,
 	strictSchema: false,
 	strict: false,
 	messages: true,
 } );
 
-addFormats( ajv );
+addFormats( ajv, {
+	mode: 'fast',
+	formats: [ 'date', 'time', 'email', 'uri' ],
```

Comment:
right now those are the only validation we will support, we can support more later but didn't see a reason for.